docs: PE-5647: Allow skipping of node drains when running cluster upgrades for single-node clusters. (#4851)
spectrocloud/librarium: docs/docs-content/clusters/edge/cluster-management/skip-draining.md

   :::

5. Create a new cluster with the updated cluster profile, or update an existing cluster to use the new cluster profile.
   For more information, refer to [Create a Cluster](../site-deployment/cluster-deployment.md) or
   [Update a Cluster](../../cluster-management/cluster-updates.md).

   For existing clusters, when you update the profile without changing the `system.uri` parameter, these changes alone
   will take effect and will not trigger any reboot or repave. The next time you make a change to the cluster profile
   that will trigger a repave or reboot, the new draining settings will apply. For more information about cluster
   behavior during upgrades, refer to [Edge Cluster Upgrade Behavior](../cluster-management/upgrade-behavior.md).

## Validate

1. Update the cluster to use a new version of a cluster profile. Ensure that your change will trigger a reboot or
   repave. For more information, refer to [Update a Cluster](../../cluster-management/cluster-updates.md) and
   [Edge Cluster Upgrade Behavior](../cluster-management/upgrade-behavior.md).

2. After the upgrade is completed, use `kubectl logs` to check on a node for which you skipped pod draining. Confirm

>    that you do not see any messages that look like `Evicting pod <pod-name> as part of upgrade plan`. The absence of

Author: ```suggestion
   that no messages that look like `Evicting pod <pod-name> as part of upgrade plan` are displayed. The absence of
```
